docs/README.md

```diff
@@ -0,0 +1,7 @@
+# VSCode Kafka Documentation
+
+Welcome to the [vscode-kafka](https://github.com/jlandersen/vscode-kafka) documentation.
+
+* [Kafka Explorer](Explorer.md#explorer)
+* [Producing](Producing.md#producing)
+* [Consuming](Consuming.md#consuming)

```

Comment:
Consuming messages

Reply:
I have fixed that but is not Consuming topics?

Reply:
both are probably correct. Consuming messages should be understood by most people though.

Reply:
fixed